driver-core/src/test/unit/com/mongodb/ConnectionStringTest.java

```diff
@@ -65,6 +65,6 @@ public void shouldPassAllOutcomes() {
 
     @Parameterized.Parameters(name = "{1}")
     public static Collection<Object[]> data() {
-        return JsonPoweredTestHelper.getTestData("/connection-string");
+        return JsonPoweredTestHelper.getTestData("connection-string");
```

Comment:
Do we have that ticket to delete all these test classes, and run "everything in the submodule" from one place?

Reply:
Added JAVA-5857 👍